assets/js/container/imprint/imprint.js

```diff
@@ -29,6 +29,7 @@ class Imprint extends React.Component {
               41469 Neuss</p>
             <h2>Kontakt</h2>
             <p>E-Mail: <a href="mailto: contact@print4health.org">contact@print4health.org</a></p>
+            <p>Telefon: 0160/9566 9630</p>
```

Comment:
not stressed out enough already, hm?  😉 ☎️

Reply:
was a hint from the laywer that this should be present on the imprint in order to avoid any legal trouble...